frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -1237,46 +1237,51 @@ const ManageHostsPage = ({
   const renderMDMEnrollmentFilterBlock = () => {
     if (!mdmEnrollmentStatus) return null;
 
-    let label: string;
-    switch (mdmEnrollmentStatus) {
-      case "automatic":
-        label = "MDM status: On (automatic)";
-        break;
-      case "manual":
-        label = "MDM status: On (manual)";
-        break;
-      default:
-        label = "MDM status: Off";
-    }
+    const label = `MDM status: ${invert(MDM_STATUS)[mdmEnrollmentStatus]}`;
 
     let TooltipDescription: JSX.Element;
     switch (mdmEnrollmentStatus) {
```

Comment:
I recently received this guidance from @ghernandez345 so passing that on here – I think we prefer a pattern using a separate config file containing something like 
```
mdmEnrollmentStatusText = {
    "automatic": ( <span className={`tooltip__tooltip-text`}>
           {copy copy copy}
          </span>),
      "manual": ...
}
```
and then importing that and keying into it in this file to get the text rather than having a switch statement with the content right there in the same file.

Not a dealbreaker either way

Reply:
tytyty